client/src/plugins/camunda-plugin/deployment-tool/__tests__/DeploymentToolSpec.js

```diff
@@ -190,6 +203,50 @@ describe('<DeploymentTool>', () => {
     });
 
 
+    it('should deploy with saved attachments', async () => {
```

Comment:
Maybe we should add another test case for [a file not being able to read](https://github.com/camunda/camunda-modeler/pull/2137/files#diff-b7aae43efe5f7296724b7853ad9d96f11f2aa3c709dce5d63add6f19761a99ddR365)?

Reply:
Sure, I'll do that.

Reply:
[Done](https://github.com/camunda/camunda-modeler/pull/2137/commits/3951d1db797b27ec8200c7ee0156da4cf3b9baad#diff-c0ab51f9f7aa4b702043ccd1af153350b386a56b24248476309443b0bdfe69d4R254)